requirements/base.txt

```diff
@@ -1,5 +1,5 @@
-boto3==1.6.2
-botocore==1.9.2
+boto3>=1.6.2
```

Comment:
Could you explain how this works? I tried skimming through pip's docs on this but I couldn't find any docs how this works.

My specific questions are:
* When running `pip install -r requirements.txt -r requirements-dev.txt` command, will this install the min version if boto3 is not installed?
* Is this just scoped to any minor version? So if you had boto3 2.0 installed, 1.6.2 would not be installed?

Trying to wrap my head around how this works.

Reply:
This is tricky.
I have another project that when forcing a specific boto3 version just stopped working in new installations because the Amazon Linux was coming with a new boto3 version and pip tried to install this specific version because you instructed to do that with '==' and not '>=' (equal or higher version)
It took sometime to realize the bug in my package and after that I just changed the dependency version to be "equal or higher than" instead of "equal to"
Well, I changed just boto3 because I got this bug in the past in my code but we can think about other dependencies to be "equal or higher" too.